packages/vuetify/src/composables/layout.ts

```diff
@@ -17,7 +17,14 @@ type LayoutItem = {
 }
 
 interface LayoutProvide {
-  register: (id: string, priority: Ref<number>, position: Ref<Position>, amount: Ref<number | string>) => Ref<Record<string, unknown>>
+  register: (
+    id: string,
+    priority: Ref<number>,
+    position: Ref<Position>,
+    layoutSize: Ref<number | string>,
+    itemSize: Ref<number | string>,
```

Comment:
Thoughts on `elementSize` instead of `itemSize`?

Reply:
I have none.

Reply:
do we use `element` anywhere now? we have list item, tab item, group item etc, i'd keep it consistent, `element` could be used only for html elements